wpilibjExamples/examples.xml

```diff
@@ -251,6 +251,45 @@
             <file source="examples/gearsbot/subsystems/Wrist.java" destination="src/$package-dir/subsystems/Wrist.java" />
         </files>
     </example>
+    <example>
+        <name>PacGoat</name>
+        <description>A fully functional example CommandBased program for FRC Team 190&amp;#39;s 2014
```

Comment:
The `&amp;#39;` part doesn't look right.. should it just be `'`?

Reply:
It was displayed correctly in eclipse, but I simplified it.

Reply:
Does it still display correctly in eclipse?

Reply:
yes